src/API.js

```diff
@@ -0,0 +1,34 @@
+const nameInput = document.getElementById('inputKey');
+const scoreInput = document.getElementById('inputValue');
+
+const sendData = () => {
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/JISCZSyYfIzlC6N08Rja/scores/', {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: 'God of war' }),
+  });
+};
+sendData();
+
+const sendScore = () => {
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/JISCZSyYfIzlC6N08Rja/scores/', {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: nameInput.value, score: scoreInput.value }),
+  });
+};
```

Comment:
- [ ] Great job implementing the fetch leaderboard, I can see all the submitted data 👏  but right now your post is not working. When I changed the parameter from `name` to `user`, it worked! you should consider double-checking it. You can take a look at it here:

![image](https://user-images.githubusercontent.com/66967274/146466843-641f033d-8fde-4304-a59c-fb9797597df0.png)